tools/serverpod_cli/lib/src/util/model_helper.dart

```diff
@@ -142,30 +136,40 @@ class ModelHelper {
         ));
   }
 
-  static List<String> extractPathFromConfig(ModelLoadConfig config, Uri uri) {
-    List<List<String>> paths = [
+  static List<String> extractPathFromConfig(
```

Comment:
Comment: Just a note, do we have tests for this?

Reply:
Yes, these are covered in `model_helper_test` as well as in the model file load tests (that use this internally).